rx-preferences/src/main/java/rx/android/preferences/ObjectPreference.java

```diff
@@ -35,9 +35,9 @@ public ObjectPreference(SharedPreferences sharedPreferences, Converter<T> conver
 
   @Override public void set(T value) {
     try {
+      this.value = value;
```

Comment:
Let's move this to after the next line - otherwise when the converter fails we'll still incorrectly hold on to the new value.